synapse/lib/cell.py

```diff
@@ -2640,6 +2640,17 @@ async def setHttpSessInfo(self, iden, name, valu):
         if sess is not None:
             sess.info[name] = valu
 
+    @s_nexus.Pusher.onPushAuto('http:sess:set:vals')
+    async def setHttpSessInfoVals(self, iden, vals: dict):
+        for name, valu in vals.items():
+            s_msgpack.isok(valu)
+        for name, valu in vals.items():
+            self.sessstor.set(iden, name, valu)
+        sess = self.sessions.get(iden)
+        for name, valu in vals.items():
+            sess.info[name] = valu
+
+    setHttpSessInfo
```

Comment:
extra string?

Reply:
Yeah..I would've expected that to throw a SyntaxError :(